src/App.js

```diff
@@ -0,0 +1,63 @@
+import React, { useEffect, useState } from "react";
+import Card from "./Card";
+import Header from "./Header";
+import Footer from "./footer";
+import "./App.css";
+
+function App() {
+  const [number, setNumbers] = useState([]);
+
+  const deleteFromNumer = (index) => {
+    const newArr = [...number];
+    newArr.splice(index, 1);
+    setNumbers(newArr);
```

Comment:
create new function where new array will be set and localstorage updated to not set localstorage item in every function. Use the function everywhere